### DISCUSSION TITLE
[Projects Beta] Task Custom Field Changes Reflected in Issue Log

### DISCUSSION THREAD

**Discussion by RickCogley** [community/community#6426] [Projects and Issues]
When you make changes in a normal Issue, they are reflected in the issue log in the main panel. It would be great for the new project tasks that you can link to an issue, if there are custom fields on the task that get changed, for those changes to be optionally reflected in the Issue change log. For clarity, I'm talking about where it says the assignee was changed, or the issue was closed and so on. 

<img width="960" alt="image" src="https://user-images.githubusercontent.com/512328/137605027-78c8bc2e-3b9a-4688-8917-d9433268a22b.png">

**Comment by abaldwinls** [comment]
Yes please! We use the custom fields for sprint tracking, story pointing, etc. We have no visibility if someone changes these fields.  This would be very helpful for my team
